documentation/docs/get-started/intro.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 1
+slug: /get-started
+---
+
+# Intro
+
+Let's discover **VCKit in less than 10 minutes**.
```

Comment:
Should the intro of get-started have the targets after reading through? Like
- Create VC
- Revoke VC
- Verify VC

Reply:
I updated this